examples/misc/bin/try_dart/classes.dart

```diff
@@ -0,0 +1,41 @@
+// ignore_for_file: annotate_overrides, sort_constructors_first, type_annotate_public_apis
+// Abstract classes can't be instantiated.
+// ignore: one_member_abstracts
+abstract class Item {
+  use();
+}
+
+// Classes can implement other classes.
+class Chest<T> implements Item {
+  List<T> contents;
+
+  // Constructors can assign arguments to instance variables using `this`.
+  Chest(this.contents);
+
+  use() => print("$this has ${contents.length} items.");
+}
+
+class Sword implements Item {
+  int get damage => 5;
+
+  use() => print("$this dealt $damage damage.");
+}
+
+// Classes can extend other classes.
+class DiamondSword extends Sword {
+  int get damage => 50;
```

Comment:
The original source declared Sword.damage and DiamondSword.damage as a field. The analyzer (rightly) complained that we shouldn't override fields: `Don't override fields`. So I change the fields to getters. Ok?

Reply:
Can we move the updating of the examples to a separate PR, please?

Reply:
Yes, will do.

Reply:
Code change undone.

Reply:
I've created #2126 to track this, and offer a place for discussion.